test/unit/lodash.spec.js

```diff
@@ -0,0 +1,628 @@
+                expect(fun(NaN)).to.eql(false);
+            });
+        });
+        
+        describe('overEvery2', function() {
+            var params,
+                elem1;
+            before(function() {
+                params = datasets.overEvery2();
+                elem1 = params[0];
+            });
+            it('should check type', function() {
+                expect(elem1 instanceof Function).to.eql(true);
+            });
+            it('should return boolean from given class', function(){
+                var fun = _.overEvery.apply(_, params);
+                expect(fun('lodash')).to.eql(true);
+                expect(fun(undefined)).to.eql(false);
+            });
+        });
+
+        describe('overEvery3', function() {
+            var params,
+                elem1;
+            before(function() {
+                params = datasets.overEvery3();
+                elem1 = params[0];
+            });
+            it('should check type', function() {
+                //expect(elem1 instanceof Array).to.eql(true);
+                expect(elem1 instanceof Function).to.eql(true);
+                expect(elem1 instanceof Function).to.eql(true);
+            });
+            it('should return boolean from given predicate', function(){
+                var fun = _.overEvery.apply(_, params);
+                expect(fun(7)).to.eql(true);
+                expect(fun('4')).to.eql(true);
+                expect(fun('five')).to.eql(false);
+                expect(fun(null)).to.eql(false);
+            });
+        });
+    });
+    describe('overSome', function() {
+        describe('overSome1', function() {
+            var params,
+                elem1,
+                elem2;
+            before(function() {
+                params = datasets.overSome1();
+                elem1 = params[0];
+                elem2 = params[1];
+            });
+            it('should check type', function() {
+                expect(elem1 instanceof Function).to.eql(true);
+                expect(elem2 instanceof Function).to.eql(true);
+            });
+            it('should return array with finite values and boolean', function(){
+                var fun = _.overSome.apply(_, params);
+                expect(fun(5)).to.eql(true);
+                expect(fun('4')).to.eql(true);
+            });
+        });
+
+        describe('overSome2', function() {
+            var params,
+                elem1;
+            before(function() {
+                params = datasets.overSome2();
+                elem1 = params[0];
+            });
+            it('should check type', function() {
+                expect(elem1 instanceof Function).to.eql(true);
+            });
+            it('should return array with String class values', function(){
+                var fun = _.overSome.apply(_, params);
+                expect(fun(/\d.\w/i)).to.eql(true);
+                expect(fun('number')).to.eql(true);
```

Comment:
You should check also false value. Now user can pass function which always return true.